src/styles/AddProperty.css

```diff
@@ -0,0 +1,43 @@
+h1 {
+    margin-top: 10%;
+   color: rgb(211, 211, 211);
+}
+
+.add-property {
+    margin: auto;
+    width: 95%;
+    text-decoration: none;
+    background-color: #c5c5c5;
+    background-image: linear-gradient(to left, rgba(255, 255, 255, 0.637), rgba(0, 0, 0, 0.648)), url('../icons/big-tiny-belly-XtnNrQYC7ts-unsplash.jpg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: 0px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.add-property__form {
+    margin: 30px;
+    margin-top: 20px;
+    border-radius: 6px;
+    padding: 20px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+   
```

Comment:
Should never leave any clear lines inside a CSS class definition, theres another of these on line 42 of this file too